Heading/Heading.module.less

```diff
@@ -5,7 +5,15 @@
 @import "../styles/skin.less";
 
 .heading {
+	.backButton {
+		align-items: center;
+		display: flex;
+		justify-content: center;
```

Comment:
`justify-content` This property isn't needed. Is this necessary?

Reply:
I will remove it. Thanks!